packages/common/pipes/file/parse-file-options.interface.ts

```diff
@@ -5,4 +5,9 @@ export interface ParseFileOptions {
   validators?: FileValidator[];
   errorHttpStatusCode?: ErrorHttpStatusCode;
   exceptionFactory?: (error: string) => any;
+
+  /**
+   * Defines if file parameter is optional. Default is false.
+   */
+  fileIsOptional?: boolean;
```

Comment:
what about

```suggestion
  required?: boolean;
```

and then the default would `true`.